scripts/02248-Olanre.js

```diff
@@ -0,0 +1,10 @@
+let fullName = "Ojo Cornelius Lanre";
+let ID = "HNG-02248";
+let language = "Javascript";
+
+function hngTask() {
+	let task = `Hello World, this is ${fullName} with HNGi7 ID ${ID} using ${language} for stage 2 task`;
```

Comment:
Please include your email after the word `task` and a space. Look [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700) for details